server/database/database.js

```diff
@@ -0,0 +1,47 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const {
+  PGUSER, PGHOST, PGPASSWORD, PGDATABASE, PGPORT,
+} = process.env;
+
+const pool = new Pool({
+  user: PGUSER,
+  host: PGHOST,
+  database: PGDATABASE,
+  password: PGPASSWORD,
+  port: PGPORT,
+});
+
+class Database {
+  constructor(table) {
+    this.table = table;
+  }
+
+  async queryBuilder(query, params) {
+    try {
+      const client = await pool.connect();
+      const results = await client.query(query, params);
+      client.end();
+      return { results, errors: null };
+    } catch (errors) {
+      return { results: null, errors };
+    }
+  }
+
+  async all() {}
```

Comment:
What is the use of this empty functions?

Reply:
i will use them as I'm working on the features but in this RP they don't play any role

Reply:
Then I do not see any reason as to why we have them here, we could remove them until we need them